README.md

```diff
@@ -302,7 +302,8 @@ Thanks to all [contributors](https://github.com/thangchung/awesome-dotnet-core/g
 * [Polly](https://github.com/App-vNext/Polly) - .NET 3.5 / 4.0 / 4.5 / PCL library that allows developers to express transient exception and fault handling policies such as Retry, Retry Forever, Wait and Retry or Circuit Breaker in a fluent manner.
 
 ### E-Commerce and Payments
-* [nopCommerce](https://github.com/nopSolutions/nopCommerce) - Free open-source ecommerce shopping cart (ASP.NET MVC / ASP.NET Core MVC )
+* [nopCommerce](https://github.com/nopSolutions/nopCommerce) - Free open-source ecommerce shopping cart (ASP.NET MVC / ASP.NET Core MVC ).
```

Comment:
why do you modify this one please revert it. Thank you

Reply:
I just added full stop there. isn't that necessary ?

Reply:
Sorry I dont understand what you mean ? I think you switch to this branch when you are in nopCommerce branch so that why you get this issue

Reply:
is it okay to delete this pull request and please  add PayPal ?

Reply:
you can create another pr then I will merge for you